tests/helpers.rs

```diff
@@ -0,0 +1,111 @@
+//! Helpers for tests which exercise the overall application, in particular
+//! the `MDBook` initialization and build/rendering process.
+//!
+//! This will create an entire book in a temporary directory using some
+//! dummy content.
+
+#![allow(dead_code, unused_variables, unused_imports)]
+extern crate tempdir;
+
+use std::path::Path;
+use std::fs::{self, File};
+use std::io::{Read, Write};
+
+use tempdir::TempDir;
+
+
+const SUMMARY_MD: &'static str = "# Summary
```

Comment:
Actually, since you convert to bytes later you can directly use `ìnclude_bytes!()`